rustfmt-core/rustfmt-lib/src/test/mod.rs

```diff
@@ -385,7 +389,8 @@ fn self_tests() {
     if !is_nightly_channel!() {
         return;
     }
-    let mut files = get_test_files(Path::new("tests"), false);
+    let skip_file_white_list = ["target", "tests"];
+    let mut files = get_test_files(Path::new("../../rustfmt-core"), true, &skip_file_white_list);
     files.push(PathBuf::from("src/lib.rs"));
```

Comment:
So I was also originally thinking that we could restore `self_tests` by conditionally enabling the `recursive` Config setting that could be controlled via additional flags to `check_files()` and then to `idempotent_check()` vs. running in non-recursive mode and collecting each individual file. 

However, if we are going to use the updated form of `get_test_files` to collect every file, then I don't think there's a need to have this line to push the entry `lib.rs` file again (it should already be there), right?

Reply:
Certainly. I fixed it. Thank you for the review.